src/content/getting-started/prepare-your-provider-account/vmware-vsphere/_index.md

```diff
@@ -3,11 +3,103 @@ title: Prepare your provider account for VMware vSphere
+
+A vSphere environment has no concept of the load balancer, which Kubernetes requires to expose services of the type load balancer and the API in a highly available mode. As a result, the Cluster API implementation includes `kube-vip`, a layer-2 load balancer to address all environments. The other option is to use NSX Advanced Load Balancer when available in your environment.
+
+{{< tabs >}}
+{{< tab id="flags-kubevip" title="kube-vip">}}
+Since vSphere has no concept of load balancers out of the box, Cluster API ships with [kube-vip]({{< relref "/vintage/advanced/cluster-management/vsphere-kubevip" >}}), a layer two load balancer that works with [ARP](https://en.wikipedia.org/wiki/Address_Resolution_Protocol) requests. By default, `kube-vip` only handles the Kubernetes API access. Still, at Giant Swarm, we also deploy the `kube-vip` provider to offer the capability to create services of type load balancer.
+
+The ARP layer two protocol informs the network of the location of a new host address. `kube-vip` runs in-cluster as opposed to a more traditional external load-balancer that will forward IP packets to its upstream servers.
+
+![CAPV kube-vip](capv-kubevip-excalidraw.png)
+
+Due to the in-cluster operation of `kube-vip`, the cluster network where this component is deployed must have a dedicated subnet range outside of the DHCP scope. To avoid IP conflicts, we recommend having one subnet per management cluster.
+
+![CAPV kube-vip IPAM](capv-kubevip-ipam-excalidraw.png)
+
+When deploying a Cluster API cluster, it automatically selects an IP from the IP pool by default. However, to have available IPs for services of type load balancer in the workload cluster, you must explicitly set a CIDR in the nodes' subnet.
+
+Learn more about how to configure `kube-vip` in the [advanced documentation]({{< relref "/vintage/advanced/cluster-management/vsphere-kubevip" >}}).
+{{< /tab >}}
+
+{{< tab id="flags-nsxalb" title="NSX ALB">}}
+When using NSX Advanced Load Balancer (NSX ALB), [there are several components](https://docs.vmware.com/en/VMware-vSphere/7.0/vmware-vsphere-with-tanzu/GUID-A247F5F2-AC7E-48E7-B615-F8D361C7292A.html) involved to enable load balancer capabilities in Kubernetes.
+
+The `controller` in NSX ALB plays a pivotal role. It's responsible for communicating the operations requested to the vCenter Server, ensuring the smooth functioning of the load balancer. Additionally, there is a `service engine` to manage virtual IP addresses and a Kubernetes `operator` to reconcile the NSX ALB resources in the clusters.
+
+![CAPV kube-vip](capv-nsxalb-excalidraw.png)
+
+{{< /tab >}}
+{{< /tabs >}}
+
+__Note__: The cluster nodes must have access to the vCenter endpoint on port 443 so that the controllers can manage the cluster's lifecycle.
```

Comment:
```suggestion
Network traffic requirements. The cluster nodes must have access to:

- vCenter endpoint on port 443 so that the controllers can manage the cluster's lifecycle.
- Internet on port 443 to pull artifacts from Github, download CVE databases, pull container images, etc. We support authenticated HTTP proxies and we provide a [domain allowlist](https://docs.giantswarm.io/vintage/platform-overview/security/cluster-security/domain-allowlist/) for firewall whitelisting.
```

The link points to vintage so it might need fixing.